src/main/java/ui/issuepanel/IssuePanel.java

```diff
@@ -36,7 +37,11 @@
         new KeyCodeCombination(KeyCode.N, KeyCombination.CONTROL_DOWN);
     private final KeyCombination defaultSizeWindow =
         new KeyCodeCombination(KeyCode.D, KeyCombination.CONTROL_DOWN);
+
+    // At the moment, apparently, whenever the screen refreshes, all previously lit up comments will
+    // lose the highlight.
```

Comment:
Is this PR supposed to fix that problem or is this just an observation? In other words, what's the purpose of this comment?